src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -95,7 +101,8 @@ const SubjectsStep = ({ stepLabel, btnsBox }) => {
           <AsyncAutocomplete
             labelField='name'
             onChange={onChangeCategory}
-            service={categoryService.getCategoriesNames}
+            onFocus={handleCategoryFetch}
```

Comment:
for category just use fetchOnFocus props

Reply:
just add prop it will be counted as true

Reply:
useState will be redundant

Reply:
> for category just use fetchOnFocus props

fixed it